src/components/fields/Percentage/BasicCell.tsx

```diff
@@ -1,41 +1,23 @@
 import { IBasicCellProps } from "@src/components/fields/types";
 
 import { useTheme } from "@mui/material";
-import { resultColorsScale } from "@src/utils/color";
 
 export default function Percentage({ value }: IBasicCellProps) {
   const theme = useTheme();
 
-  if (typeof value === "number")
-    return (
-      <>
-        <div
-          style={{
-            backgroundColor: resultColorsScale(value).toHex(),
-
-            position: "absolute",
-            top: 0,
-            right: 0,
-            bottom: 0,
-            left: 0,
-            opacity: 0.5,
-
-            zIndex: 0,
-          }}
-        />
-        <div
-          style={{
-            textAlign: "right",
-            color: theme.palette.text.primary,
-
-            position: "relative",
-            zIndex: 1,
-          }}
-        >
-          {Math.round(value * 100)}%
-        </div>
-      </>
-    );
-
-  return null;
+  const percentage = typeof value === "number" ? value : 0;
+  return (
+    <>
+      <div
+        style={{
+          textAlign: "right",
+          color: theme.palette.text.primary,
+          position: "relative",
+          zIndex: 1,
+        }}
+      >
+        {Math.round(percentage * 100)}%
+      </div>
+    </>
```

Comment:
```suggestion
      <div
        style={{
          textAlign: "right",
          color: theme.palette.text.primary,
          position: "relative",
          zIndex: 1,
        }}
      >
        {Math.round(percentage * 100)}%
      </div>
```

You don’t need to wrap this in a Fragment